tests/ops/util/test_connection_type.py

```diff
@@ -38,3 +41,205 @@ def test_get_connection_types():
+                DOORDASH,
+                STRIPE,
+                ZENDESK,
+                SEGMENT,
+                ConnectionType.attentive.value,
+            ],
+        ),
+        (
+            [ActionType.access],
+            [
+                ConnectionType.postgres.value,
+                ConnectionType.manual_webhook.value,
+                DOORDASH,
+                SEGMENT,
+                STRIPE,
+                ZENDESK,
+            ],
+            [
+                GOOGLE_ANALYTICS,
+                MAILCHIMP_TRANSACTIONAL,
+                ConnectionType.sovrn.value,
+                ConnectionType.attentive.value,
+            ],
+        ),
+        (
+            [ActionType.erasure],
+            [
+                ConnectionType.postgres.value,
+                SEGMENT,  # segment has DPR so it is an erasure
+                STRIPE,
+                ZENDESK,
+                ConnectionType.attentive.value,
+            ],
+            [
+                GOOGLE_ANALYTICS,
+                MAILCHIMP_TRANSACTIONAL,
+                ConnectionType.manual_webhook.value,  # manual webhook is not erasure
+                DOORDASH,  # doordash does not have erasures
+                ConnectionType.sovrn.value,
+            ],
+        ),
+        (
+            [ActionType.consent, ActionType.access],
+            [
+                GOOGLE_ANALYTICS,
+                MAILCHIMP_TRANSACTIONAL,
+                ConnectionType.sovrn.value,
+                ConnectionType.postgres.value,
+                ConnectionType.manual_webhook.value,
+                DOORDASH,
+                SEGMENT,
+                STRIPE,
+                ZENDESK,
+            ],
+            [
+                ConnectionType.attentive.value,
+            ],
+        ),
+        (
+            [ActionType.consent, ActionType.erasure],
+            [
+                GOOGLE_ANALYTICS,
+                MAILCHIMP_TRANSACTIONAL,
+                ConnectionType.sovrn.value,
+                ConnectionType.postgres.value,
+                SEGMENT,  # segment has DPR so it is an erasure
+                STRIPE,
+                ZENDESK,
+                ConnectionType.attentive.value,
+            ],
+            [
+                ConnectionType.manual_webhook.value,  # manual webhook is not erasure
+                DOORDASH,  # doordash does not have erasures
+            ],
+        ),
+        (
+            [ActionType.access, ActionType.erasure],
+            [
+                ConnectionType.postgres.value,
+                ConnectionType.manual_webhook.value,
+                DOORDASH,
+                SEGMENT,
+                STRIPE,
+                ZENDESK,
+                ConnectionType.attentive.value,
+            ],
+            [
+                GOOGLE_ANALYTICS,
+                MAILCHIMP_TRANSACTIONAL,
+                ConnectionType.sovrn.value,
+            ],
+        ),
+    ],
+)
+def test_get_connection_types_action_type_filter(
```

Comment:
i definitely need to add test coverage around the endpoint itself rather than just the underlying util, but wanted to get this up as a draft since i think it demonstrates the behavior.

if the behavior looks good, then i'll flesh out the test coverage around the endpoint/route.

Reply:
These tests are great Adam